workflows/imagery/standardising.yaml

```
@@ -48,6 +48,117 @@ spec:
         value: "*Region/District/City* *GSD* *Urban/Rural* Aerial Photos (*Year-Year*)"
       - name: description
         value: "Orthophotography within the *Region Name* region captured in the *Year*-*Year* flying season."
+      - name: producer
+        value: "AAM NZ"
```

```suggestion
        value: "Unknown"
```
Let's add an Unknown to the enum and make that the default